yml/apdb.yaml

```diff
@@ -2156,6 +2156,20 @@ tables:
     nullable: false
     description: Id of the DiaObject that this DiaForcedSource was associated with.
     ivoa:ucd: meta.id;src
+  - name: ra
+    "@id": "#DiaForcedSource.ra"
+    datatype: double
+    nullable: false
+    description: Right ascension coordinate of the position of the object at time radecMjdTai.
```

Comment:
This and the below should read `midpointMjdTai` (`radecMjdTai` is used for diaObjects)

Reply:
At least... I think that's the case? Or is this really the diaObject coordinate at the diaObject timestamp? Be extra explicit either way, since it's confusing to refer to a time that is not in this table's schema.

Reply:
I changed the wording to make it explicit that the reference is the DiaObject time.